### PR TITLE
Tracks are sorted in descending order and number added on thumbnail for better navigation

### PR DESCRIPTION
**Pull request by mvp5464**
1. Shorted all tracks in descending order so that the all tracks are easily found.
2. Added number on the track card for better navigation.

Solved #178 

ScreenshotTrack

- It solves this problem.

[Screencast1.webm](https://github.com/code100x/daily-code/assets/28997328/be86f63c-191f-48a9-88b0-acdfcdec9606)

### CODE REVIEW

**Comment by mvp5464** [comment]
It looks like need to edit/update the Date in createdAt  for some track (as they looks wrong)

**Comment by mvp5464** [comment]
@hkirat sir, should I update the PR or It's not needed now

File: apps/web/screens/Landing.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 import Link from "next/link";
 import { AppbarClient } from "../components/AppbarClient";
-import { getAllCategories, getAllTracks } from "../components/utils";
+import { getAllCategories, getAllTracks, getSecond } from "../components/utils";
 import { Categories, Tracks } from "@repo/ui/components";
 
 export async function Landing() {
-  const tracks = await getAllTracks();
+  const tracks1 = await getAllTracks();
+  const tracks = await getSecond(tracks1);
```
Comment (hkirat) [inline apps/web/screens/Landing.tsx:9]: This should be in the DB cal
also always avoid variables like tracks1

Comment (mvp5464) [reply apps/web/screens/Landing.tsx:9]: @hkirat Sir, done in PR #404

**Pull request closed without merging**